RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -22,10 +22,8 @@ extension TransactionReview.State {
 		.init(
 			message: {
 				// TODO: handle the rest of types
-				if case let .plainText(value) = message,
-				   case let .str(str) = value.message
-				{
-					return str
+				if case let .plainText(value) = message {
```

Comment:
Message does not hold an inner `PlainText`type anymore, instead `String` directly. Sargon will handle MimeType for wallets.